packages/wmr/src/lib/acorn-traverse.js

```diff
@@ -357,10 +388,21 @@ class Path {
 		if (this._regenerateParent()) {
 			this._hasString = false;
 		} else {
+			// Skip string generate optimizations as node positions won't
+			// match anymore.
+			this.ctx.forceRegenerate = true;
```

Comment:
Will this skip string optimizations on every replaceWith() call? Or only when encountering a template?

Reply:
This does skip every string optimization. The only other option I'm aware of would be to traverse the return value of `template/template.ast/t.clone()` and delete position data on every node.

Reply:
Alright went with clearing the node positions instead to keep the string optimization.